domain/domain_test.bzl

```diff
@@ -0,0 +1,14 @@
+load("//:oppia_android_test.bzl", "oppia_android_test")
+
+def domain_test(name, src, test_class):
```

Comment:
Optional, but we could probably collapse these three into 1 (e.g. just path & compute the test class & name from it).

Reply:
True, but if is alright with you, I would like to keep them separate for the sake of saving time. I also think it might make it simpler to remove these temporary files in Stage 2 if they are separate and only impact one module's tests.

Reply:
Yup--this seems fine, and I agree with the stage 2 reasoning. This will be much easier to replace with the future final solution (my intent was actually for you to do this macro change when it was suggested to safe time later when you added & the test targets, but now that most are introduced there isn't much of a win at this point).